pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -160,8 +154,8 @@ func (a *Actuator) updateMachineProviderConditions(machine *clusterv1.Machine, c
 
 // CreateMachine starts a new AWS instance as described by the cluster and machine resources
 func (a *Actuator) CreateMachine(cluster *clusterv1.Cluster, machine *clusterv1.Machine) (*ec2.Instance, error) {
-	machineProviderConfig, err := ProviderConfigFromMachine(machine)
```

Comment:
You'll need to update the APIVersion: "awsproviderconfig.k8s.io/v1alpha1", Kind: "AWSMachineProviderConfig", on the installer first for using the codec here https://github.com/openshift/installer/blob/master/pkg/asset/machines/aws/machines.go#L80

Reply:
That actually needs to be fixed in the installer.